jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -106,3 +128,139 @@ async def coroutine():
+            "refresh_token",
+            "client_credentials",
+            "urn:ietf:params:oauth:grant-type:device_code"
+          ],
+          "userinfo_signed_response_alg": "RS256",
+          "access_token_lifetime": 2592000,
+          "scope": "jans_stat offline_access profile email openid https://jans.io/auth/ssa.admin",
+          "run_introspection_script_before_jwt_creation": False,
+          "update_token_script_dns": ["inum=2D3E.5A04,ou=scripts,o=jans"]
+        }
+
+        async def coroutine():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `coroutine`

Add return type annotation: `None`

(ANN202)